salesagility/SuiteCRM-Core
# Fix #61 and #81 - Allow user to see dates in their time and format

Fix for #81 and #61 

## Description




This fix is for #61 and #81 

Updated luxon package.

## Motivation and Context

This change now allows users to change timezone and format without any issues

## How To Test This


1. Go to Calls/ Tasks.
2. Create a record and see that the date picker, field value are the same and are as expected.

## Types of changes


### Final checklist
